modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/test/java/com/liferay/portal/search/tuning/rankings/web/internal/index/name/RankingIndexNameBuilderImplTest.java

```diff
@@ -34,23 +35,25 @@ public class RankingIndexNameBuilderImplTest {
 
 	@Test
 	public void testMultiTenancy() {
-		_assertIndexName(
-			2021, companyId -> "liferay-" + companyId,
-			"liferay-2021-search-tuning-rankings");
-	}
-
-	private void _assertIndexName(
-		long companyId, IndexNameBuilder indexNameBuilder, String expected) {
-
 		RankingIndexNameBuilderImpl rankingIndexNameBuilderImpl =
 			new RankingIndexNameBuilderImpl();
 
-		rankingIndexNameBuilderImpl.setIndexNameBuilder(indexNameBuilder);
+		ReflectionTestUtil.setFieldValue(
```

Comment:
@W2jx In this commit, you fixed the usage in `_assertIndexName` and then inlined `_assertIndexName` into `testMultiTenancy`. The former change is "fix test" while the latter change is "SF". So you should do it in 2 commits.